code/mathematical_algorithms/src/exponentiation_power/exponentiation_by_squaring/exponentiation_by_squaring.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+
+int 
+power(int num, int exp)
+{
+	if(exp == 0)
```

Comment:
If, else, for, while, and switch statements should be followed by a space.

Reply:
Thanks for the suggestion! :) I've made the changes.